androidApp/src/main/java/press/editor/EditorActivity.kt

```diff
@@ -97,7 +97,7 @@ class EditorActivity : ThemeAwareActivity() {
       return Uuid.parse(intent.getStringExtra(KEY_NOTE_ID)!!)!!
     }
 
-    private fun intent(context: Context): Intent {
+    fun intent(context: Context): Intent {
       return Intent(context, EditorActivity::class.java).apply {
         putExtra(KEY_NOTE_ID, uuid4().toString())
```

Comment:
Can't we move the `uuid4().toString()` to be a activity variable? That way we can use the `EditorActivity` intent directly in the `shortcuts.xml`, instead of a separate activity to launch the `EditorActivity` wdyt?

Reply:
We need a `uuid` that doesn't change across activity recreations. If we store this as a variable, we'll have to handle saving/restoring it. Do you think that's simpler?

Reply:
Yeah, we don't need a new activity that will just launch `EditorActivity`. Should I make the change, or just address the PR feedback for now?

Reply:
On a second thought, I feel that this is fine. Don't worry. It's slightly verbose but I feel we might need this activity in the future for other purposes.